packages/operator/pkg/repository/connection/kubernetes/connection.go

```diff
@@ -206,6 +207,7 @@ func (kc *k8sConnectionRepository) CreateConnection(connection *connection.Conne
 			Labels:    transformToLabels(connection),
 		},
 		Spec: connection.Spec,
+		Status: connection.Status,
```

Comment:
We should not allow a user to set the status. This remark is true for all kubernetes repositories.